### ISSUE TITLE
[SpongeForge] void tear NBT breaks when I die with inventorykeep TRUE

### ISSUE THREAD

**Issue by Twisted-Code** [P3pp3rF1y/Reliquary#242]
first and foremost, this only seems to happen on my spongeforge server (or at least I haven't seen it in single player. It might still happen on standard forge servers), so it could be sponge's fault.
I keep losing whatever I have stored in my void tear. I think it is happening when I die and respawn, but I haven't been able to reproduce it intentionally. inventorykeep being enabled might also have something to do with it, but I have my suspicions the same would happen if I were using a Graves mod. 

If there's anything you think you might be able to use to help track down the issue, just name it and I'll try toprovide it. I really like these tears, so would be a shame to have to not use them or be limited to using them on cheap items like cobblestone 😦

**Comment by Twisted-Code** [comment]
Haven't seen this happen in a few days. maybe I'm just going crazy. IDK.

**Comment by P3pp3rF1y** [comment]
As I commented on CF, I believe you have either tried storing this in one of the storage systems which doesn't support capability NBT or ran into the forge bug which occasionally deleted NBT data (https://github.com/MinecraftForge/MinecraftForge/issues/3682). I am going to replace capability NBT on void tears with regular NBT again so that these can be safely stored in storage systems.

**Comment by Twisted-Code** [comment]
@P3pp3rF1y yeah, I'm pretty sure it was that forge bug. I noticed it again just now, and think it might have something to do with me switching between dimensions. I was just working on my nether-portal linking and noticed my cobble tear broke again.
As far as switching back to regular NBT, that's probably for the better, at least until Forge patches that bug and the capability NBT is more widely supported by storage systems

**Comment by Twisted-Code** [comment]
mind giving me a heads up when you push the change to curseforge? Because I really like these tears, when they work 😃

**Comment by Twisted-Code** [comment]
and yeah, it just happened again when switching dimensions. edit: strike that, it's fine. the tooltip just isn't showing unless it's on my hotbar

**Comment by Twisted-Code** [comment]
@P3pp3rF1y you may want to also switch the Hunter's Handgun to normal NBT. I keep finding mine empty when I know I had shots left before. In my case, it's not too big a deal since I'm just using seeking shots and have plenty of lapis, gunpowder, and gold, but it might be more than slightly annoying if someone was losing potion-imbued shots.

**Comment by P3pp3rF1y** [comment]
yeah, when I say that I want to switch to regular NBT that is for every single item. In case of hunter's handgun and rending gale it's a little more complicated and I may need to wait for a PR that I have submitted against forge, but I will convert all of these to regular nbt

**Comment by Twisted-Code** [comment]
@P3pp3rF1y any luck getting these items patched? Really want to use a void tear for my cobblestone and dirt, but I'm afraid of it resetting. As I said before, please let me know when you push a patch fixing these items, and in particular the void tears

**Comment by P3pp3rF1y** [comment]
Sorry about delayed response on this. I have actually decided against pulling capabilities from void tears and other stuff after seeing that most mods support them properly. 
Also it looks like AE supports these in 1.12 as well so at the moment I am not aware of any mod that would be causing capability data deletion.
I have also tested keep inventory in 1.12 and that seems to retain void tear contents as well.
